### ISSUE TITLE
Single-lobed right lung

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7087]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
A developmental defect of pulmonary lobation characterized by the presence of only one lobe of the right lung, which normally has three lobes.

**Parent term (use hpo.jax.org/app)**
Abnormal lung lobation HP:0002101

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:608978 
1/13  PMID:17853480

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
